components/fxa-client/src/http_client.rs

```diff
@@ -480,6 +480,10 @@ pub enum DeviceType {
     Desktop,
     #[serde(rename = "mobile")]
     Mobile,
+    #[serde(rename = "vr")]
+    Vr,
+    #[serde(rename = "tv")]
+    Tv,
```

Comment:
As a purely stylistic thing, are we able to make this `TV` and `VR` since they're proper acronyms rather than plain nouns? I don't feel strongly about it but I'm probably going to twitch a little each time I see in this variant...